sourcecode/apis/contentauthor/app/Http/Controllers/ContentAssetController.php

```diff
@@ -23,10 +23,10 @@ public function __invoke($path, Request $request)
         if (!$this->contentAuthorStorage->getBucketDisk()->exists($path)) {
             throw new NotFoundHttpException('File not found');
         }
-        
+
         $detector = new FinfoMimeTypeDetector();
         $response = new StreamedResponse;
-        $filename = basename($path);
+        $filename = preg_replace('/[\x00-\x1F\x80-\xFF]/', '', basename($path));
```

Comment:
Should include \x7f too, as this is a control character (DEL).

Reply:
so `/[\x00-\x1F\x80-\xFF-\x7f]/`?

Reply:
`/[\x00-\x1F\x7F-\xFF]/`